### DISCUSSION TITLE
Feature-Request : More Built-in starters for Radio Button

### DISCUSSION THREAD

**Discussion by si-ui** [heroui-inc/heroui#310] [Feature requests]
**Is your iteration request related to a problem?**
Would be great to have an extra option for the radio buttons - a more visual way to show them

**Describe the solution you'd like**
Would be nice to have a more visual radio button style, with control over the border colour / weight, hover style / weight etc

**Additional context**
See example here: https://mobile.twitter.com/steveschoger/status/1024720091546562560/photo/1

**Comment by jrgarciadev** [comment]
Hey @si-ui thanks for your recommendation, however, I'll convert this into a discussion since it required to create a new component joining two components `Card` and `Checkbox` which is something we have to think about
